packages/usa-header/src/styles/_usa-header.scss

```diff
@@ -150,7 +151,7 @@ $z-index-overlay: 400;
     .usa-nav {
       @include u-flex("row", "align-center", "justify-end");
       display: flex;
-      padding: 0 0 units(0.5) units(1);
+      padding: 0 0 units(1) units(1);
```

Comment:
**thought**: The clarity of this is nice. We can shorthand, but this approach seems better.